landoapi/hg.py

```diff
@@ -22,6 +22,9 @@
 landing_worker_target_ssh_port = os.environ.get("LANDING_WORKER_TARGET_SSH_PORT", "22")
 REJECTS_PATH = Path("/tmp/patch_rejects")
 
+# Name of the environment variable that will store the push user's email address.
+REQUEST_USER_ENV_VAR = "AUTOLAND_REQUEST_USER"
```

Comment:
Nice! Arguably, this minor refactor could be in a separate commit, but I understand how stack management is a lot more rough in Github-based workflows :)

Reply:
These commits will be squashed regardless. I'd say because it's part of moving everything to this module, it's also good to keep it in the same PR.